### PR TITLE
Add a caveat about how to combine styles

### PR DESCRIPTION
**Pull request by xymostech**
This comes up a lot in different discussions, and there isn't a definitive
reference for it, so I figured I should add one! No more lengthy explanations
in github issues.

I'm very open to feedback about the wording of this, please leave comments!

@jlfwong @kentcdodds @JedWatson

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,6 +248,92 @@ The generated css will be:
   }
 ```
 
+## Overriding styles
+
+When combining multiple aphrodite styles, you **must** merge all of your styles into a single call to `css()`, and **must not** combine the generated class names that aphrodite outputs (via string concatenation, `classnames`, etc.).
```
Comment (kentcdodds) [inline README.md]: Using the word `must` seems inflexible. I'd use "`should`" or "`it is recommended to`"

**Comment by kentcdodds** [comment]
LGTM (other than that one note). Thanks for adding this!

File: README.md (hunk 1 above)
Comment (jlfwong) [inline README.md]: "must" has a specific meaning in RFC style, but I agree that it might read as a little forceful here.

**Comment by JedWatson** [comment]
Looks good to me as well.

Thinking about that, `classnames` has been shockingly popular, I can imagine anyone switching to Aphrodite missing it if they're used to that way of applying conditional classes. Would you be interested in investigating a framework-specific port of it?

e.g

```jsx
import { css, classNames } from 'aphrodite'

<div className={classNames(styles.foo, { styles.bar: cond })} />
```

Would make this whole explanation friendlier ("don't use the classnames package, import it from Aphrodite" rather than "you can't do this")

**Comment by kentcdodds** [comment]
I like what @JedWatson suggests because I've often found myself having to do fancy stuff when calling `css` when the styles are conditional. This would simplify things quite a bit.

**Comment by jlfwong** [comment]
You shouldn't need to do fancy stuff, since if any of the arguments is false-y, it gets ignored.

Instead of `<div className={classNames(styles.foo, { styles.bar: cond })} />`, you can do this:

`<div className={classNames(styles.foo, cond && styles.bar })} />`

This is probably not documented in the README.md though, if someone would like to add that :)

**Comment by kentcdodds** [comment]
Would this work?

```jsx
<div className={classNames(styles.foo, cond && styles.bar, cond2 && styles.baz, cond3 && styles.foobar })} />
```

**Comment by jlfwong** [comment]
@kentcdodds whoops, sorry -- I miswrote the comment about "you shouldn't need to do fancy stuff" -- see edit. My point was that this kind of conditional application of classnames is already supported by the `css()` function. I was not discussing the semantics of a potential `classnames` function, I was discussing the existing semantics of the `css()` function.

**Comment by kentcdodds** [comment]
Haha, neither was I actually! See _my_ update :-)

**Comment by xymostech** [comment]
@kentcdodds yes, that does work! +1 to adding documentation about this, I didn't realize people didn't know this.

**Comment by jlfwong** [comment]
I think this just slipped through the cracks? LGTM to merge @xymostech

**Comment by xymostech** [comment]
(I made the wording a little less intense. Merge incoming...)

**Pull request merged**